MSAL/src/native_auth/public/MSALNativeAuthChallengeTypes.h

```diff
@@ -25,13 +25,21 @@
 #ifndef MSALNativeAuthChallengeTypes_h
 #define MSALNativeAuthChallengeTypes_h
 
-/// The set of capabilities that an application wishes to support for Native Auth operations.
-///
-/// Valid options are:
-/// * OOB: The application can support asking a user to supply a verification code that is sent by email.
-/// * Password: The application can support asking a user to supply a password
+#import <Foundation/Foundation.h>
+
+/** 
+    The set of capabilities that an application wishes to support for Native Auth operations.
+
+    Valid options are:
+    * OOB: The application can support asking a user to supply a verification code that is sent by email.
+    * Password: The application can support asking a user to supply a password
+*/
+
 typedef NS_OPTIONS(NSInteger, MSALNativeAuthChallengeTypes) {
```

Comment:
Now that I'm thinking, shouldn't this option be named as `MSALNativeAuthChallengeType` instead of `MSALNativeAuthChallengeType*S*` ?

Reply:
That is actually a Set (NS_OPTIONS) so you can pass into this init:
```
public init(
        configuration config: MSALPublicClientApplicationConfig,
        challengeTypes: MSALNativeAuthChallengeTypes) throws {
```
A set, like so:
`MSALNativeAuthPublicClientApplication(configuration: config, challengeTypes: [.OOB, .password])`
and even if it's one, it's like this:
`MSALNativeAuthPublicClientApplication(configuration: config, challengeTypes: [.OOB])`
which still signifies a Set/Array as in multiple items

Reply:
Thanks for the clarification Silviu. All good